qa/actions-run-tests.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
Why not existing qa/runtests.sh?

Reply:
it has a lot of excess stuff like setting terminal colors , preparing virtual env  etc . added this just to differentiate

Reply:
also have removed virtual env

Reply:
Can you fix this review comment?

Reply:
@sawood14012 , You can do a similar workaround done for backbone? i.e install virtualenv package.

Reply:
yup will do